src/components/Scorecards/ScorecardsPage/ScorecardList.tsx

```diff
@@ -53,15 +40,17 @@ export const ScorecardList = () => {
   const [searchQuery, setSearchQuery] = useInput();
 
   const {
-    value: scorecards = [],
-    loading,
-    error,
+    value: scorecards,
```

Comment:
made this change so the screen doesn't jerk from an empty list -> having scorecards, but rather shows a loader throughout

Reply:
Oh weird, I would've expected to show the loader given its true. Ty for fixing!! Mind doing it for initiatives too?